### DISCUSSION TITLE
Motherboard temperature

### DISCUSSION THREAD

**Discussion by FOX1185** [greghesp/ha-bambulab#808] [Ideas]
Good morning, would it be possible to obtain the temperature value of the printer's motherboard? It could be useful as a parameter to monitor, especially if you modify the printer to have the chamber heated and want to keep an eye on the temperature of the electronics without having to install a thermometer in the compartment.

**Comment by AdrianGarside** [comment]
If there is a temperature sensor specific to the motherboard, it isn't exposed in the mqtt payload.
